### PR TITLE
drakrun: fix IPT support

### PR DESCRIPTION
**Pull request by icedevml**
* remove `enable_ipt` from `config.ini`, IPT is just enabled by requesting activation of `ipt` plugin
* activating `ipt` implies activation of `codemon`
* provide appropriate switches for `codemon`

### CODE REVIEW

File: drakrun/drakrun/main.py [hunk 1]
```diff
@@ -162,6 +162,11 @@ def generate_plugin_cmdline(self, quality, enabled_plugins):
             plugin_list = self.active_plugins[quality]
 
         plugin_list = list(set(plugin_list) & set(enabled_plugins))
+
+        if "ipt" in plugin_list and "codemon" not in plugin_list:
+            self.log.info("Using ipt plugin implies using codemon")
```
Comment (kscieslinski) [inline drakrun/drakrun/main.py:167]: Can we add same check on drakcore frontend to?

**Review by kscieslinski — APPROVED** [review]

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/main.py:167]: @kscieslinski Do you mean displaying a warning box with the content as above?

Comment (kscieslinski) [reply drakrun/drakrun/main.py:167]: Yes.

Comment (icedevml) [reply drakrun/drakrun/main.py:167]: Added

image

**Pull request merged**